javaee/concurrency/src/main/java/fish/payara/examples/concurrency/lastexecution/LastExecutionRest.java

```diff
@@ -0,0 +1,46 @@
+package fish.payara.examples.concurrency.lastexecution;
+
+import jakarta.annotation.Resource;
+import jakarta.enterprise.concurrent.*;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Date;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Path("last-execution")
+public class LastExecutionRest {
```

Comment:
What if instead to create a new class you only add a new Rest service method on the original class?

Reply:
It is another possibility, but I would like to separate them for maintainability reasons.